bin/fiddle-mount.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * This module provides a function to join jsfiddle examples into single fancy html output.
+ * The examples must have the following layout:
+ *   example_name
+ *    |- demo.html
+ *    |- demo.js
+ *    \- demo.css
+ *
+ */
+const path = require('path');
+const fs =   require('fs');
+const dot =  require('dot');
+
+/*
+ * Generate html output for given example.
+ * Calls callback(contents) when the contents of the output html file is generated.
+ *
+ * If exampleName is '*' then one big file with all examples is generated.
+ *
+ */
+const parse = (exampleName, examplesPath, callback, onlyBody=false, conf=null) => {
+    const exampleDir = path.resolve(__dirname, examplesPath, exampleName);
+  
+    const globalExampleTemplateFile = fs.readFileSync(path.resolve(__dirname, examplesPath, 'index.html'));
+    const singleExampleTemplateFile = fs.readFileSync(path.resolve(__dirname, examplesPath, 'example.html'));
+
+    const globalExampleTemplate = dot.template(globalExampleTemplateFile);
+    const singleExampleTemplate = dot.template(singleExampleTemplateFile);
+
+  
+    // Make one big html out of all examples
+    if(exampleName == '*') {
+    
+        const examples = fs.readdirSync(path.resolve(__dirname, examplesPath)).filter((folder) => {
+            return fs.lstatSync(path.resolve(__dirname, examplesPath, folder)).isDirectory();
+        });
+    
+        let joinArray = [];
+    
```

Comment:
W GitHubie tego nie widać, ale jak przeglądam lokalnie to widać dużo "trailing whitespace". Włącz sobie może w swoim edytorze, żeby usuwał przy zapisywaniu.

Reply:
PS. Właśnie widzę, że w jednym z kolejnych commitów to poprawiasz ;)